src/lib/gitutils.ts

```diff
@@ -345,6 +346,27 @@ export const checkoutCommitPushCreatePRLink = async (
   }
 };
 
+/**
+ * Returns a git repository url
+ *
+ * @param opts Options object from commander.
+ * @param gitOriginUrl Git origin URL which is used to set values
+ *        for pipeline, repoName and repoUrl
+ */
+export const validateRepoUrl = (
```

Comment:
Looks good. Just need a unit test for this method

Reply:
Done!